Specs/Scene/PostProcessStageLibrarySpec.js

```diff
@@ -301,52 +305,41 @@ describe(
         new HeadingPitchRoll()
       );
 
-      const model = scene.primitives.add(
-        Model.fromGltf({
+      return loadAndZoomToModel(
+        {
           url: boxTexturedUrl,
+          incrementallyLoadTextures: false,
```

Comment:
was the `incrementallyLoadTextures` option added for any particular reason?

Reply:
I was noticing that sometimes the texture was loading in time for the test but other times I was getting the default texture. For these post-processing effects having the texture is helpful because the color values change more noticeably than the default solid white texture.

Reply:
makes sense, thanks!